vendor/bat-native-ads/src/bat/ads/internal/eligible_ads/sample_ads.h

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ELIGIBLE_ADS_SAMPLE_ADS_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ELIGIBLE_ADS_SAMPLE_ADS_H_
+
+#include <map>
+#include <ostream>
```

Comment:
Is <ostream> include needed?

Reply:
@aseren yes because of `NOTREACHED() << "Sum should always be strictly less than probability";`

Reply:
can we not include `base/notreached.h`?